### PR TITLE
Fix usage through phar://

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: bin/manala [hunk 1]
```diff
@@ -15,6 +15,8 @@ require_once __DIR__.'/../vendor/autoload.php';
 use Manala\Command;
 use Symfony\Component\Console\Application;
 
+define('MANALA_DIR', \Phar::running() ?: __DIR__.'/../');
```
Comment (ogizanagi) [inline bin/manala]: Out of curiosity: is the `\Phar::running` call needed ? Won't the path always be `__DIR__.'/../'` whether you are running the phar or the php script ?

Comment (chalasr) [reply bin/manala]: Indeed, it was some debugging leavings. Removed

File: bin/manala [hunk 2]
```diff
@@ -15,8 +15,6 @@ require_once __DIR__.'/../vendor/autoload.php';
 use Manala\Command;
 use Symfony\Component\Console\Application;
 
-define('MANALA_DIR', \Phar::running() ?: __DIR__.'/../');
```
Comment (ogizanagi) [inline bin/manala]: I would have kept the constant, though

**Review by ogizanagi — APPROVED** [review]

File: src/Config/Renderer.php [hunk 3]
```diff
@@ -31,7 +31,7 @@ public static function render(Config $config, Vars $vars)
     {
         $template = $config->getTemplate();
 
-        if (!is_writable($template) || !is_readable($template)) {
+        if (!is_readable($template)) {
             throw new \RuntimeException(sprintf('The template file "%s" is either not readable, not writable or it doesn\'t exist.', $template));
```
Comment (benji07) [inline src/Config/Renderer.php]: you should change the error message, now that you only check if the template is readable

**Review by benji07 — CHANGES_REQUESTED** [review]

File: bin/manala (hunk 2 above)
Comment (chalasr) [reply bin/manala]: Do you just find it convenient? We can live without as there is only one occurrence of `__DIR__` in `src/` for now (should not growth).

File: src/Config/Renderer.php (hunk 3 above)
Comment (chalasr) [reply src/Config/Renderer.php]: Thanks, fixed

File: bin/manala (hunk 2 above)
Comment (ogizanagi) [reply bin/manala]: Sure, that's only for convenient reasons (and preserve from any issue if the files requiring the get the project root are moved elsewhere).

But that's not a big deal actually :)

**Pull request merged**